backend/src/config/index.js

```diff
@@ -102,8 +102,8 @@ nconf.defaults({
       roleAdmin: process.env.NOMINAL_ROLL,
       rolesAllowed: process.env.NOMINAL_ROLL_ROLES ? process.env.NOMINAL_ROLL_ROLES.split(',') : '', // comma separated list
     },
-    secureMessage: {
-      // rootURL: process.env.SECURE_MESSAGE_API_URL
+    exchange: {
```

Comment:
Could we use the name `edx` instead of `exchange`?  I think we would set other endpoints in the object.